bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```
@@ -1117,12 +1118,13 @@ def cc_local_toolchain_suite(name, cpus):
             objcopy_files = ":" + name + "_empty",
             strip_files = ":" + name + "_empty",
             supports_param_files = 1,
-            toolchain_config = ":" + name + "_local_config_" + cpu,
-            toolchain_identifier = name + "_local_" + cpu,
+            toolchain_config = ":" + config_name + "_config",
+            toolchain_identifier = config_name,
+        )
+        native.toolchain(
+            name = config_name,
+            exec_compatible_with = ["@platforms//cpu:" + cpu, "@platforms//os:" + os],
+            target_compatible_with = ["@platforms//cpu:" + cpu, "@platforms//os:" + os],
```

Maybe put the labels in a local to reduce repetition?

```suggestion
        compatible_with = ["@platforms//cpu:" + cpu, "@platforms//os:" + os]
        native.toolchain(
            name = config_name,
            exec_compatible_with = compatible_with,
            target_compatible_with = compatible_with,
```